examples/kafka-authentication-sasl-plain-producer/kafka_authentication_sasl_plain_producer.out

```diff
@@ -1,7 +1,5 @@
-# To run this sample, navigate to the directory that contains the
-# `.bal` file, and execute the `bal run` command below.
 bal run kafka_authentication_sasl_plain_producer.bal
 
 # If the producer is authenticated and the message is sent successfully,
-# the following message will be printed in the console.
+# the following message will be printed in the Console.
```

Comment:
Why is this capitalized?

Reply:
removed all in [d9f6e25](https://github.com/ballerina-platform/ballerina-distribution/pull/1883/commits/d9f6e252be0d76927303cdd2f3131c7289d5ea00)